src/GirCore.sln

```diff
@@ -204,6 +204,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "img", "img", "{15AC3143-968
 		..\docs\img\GtkApp.png = ..\docs\img\GtkApp.png
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GtkSource-5", "Libs\GtkSource-5\GtkSource-5.csproj", "{9D19B388-AD8B-4DC5-B351-5D9FCEAE8FEA}"
```

Comment:
Can you please add a new solution folder named "GtkSource-5" into the Samples folder of the solution file and add your sample project to it?

In case you don't have an IDE supporting this I can add this for you.